components/operator/internal/state/controller_configuration.go

```diff
@@ -37,15 +36,6 @@ func sFnControllerConfiguration(ctx context.Context, r *reconciler, s *systemSta
 	return nextState(sFnApplyResources)
 }
 
-func warnAboutDeadFields(s *systemState) {
-	if s.instance.Spec.FunctionTimeoutSec != "" {
-		s.warningBuilder.With(functionTimeoutDepreciationMessage)
-	}
-	if s.instance.Spec.FunctionRequestBodyLimitMb != "" {
-		s.warningBuilder.With(functionRequestBodyLimitDepreciationMessage)
```

Comment:
Remove `functionRequestBodyLimitDepreciationMessage` and `functionTimeoutDepreciationMessage` from `components/operator/internal/state/controller_configuration.go`

Reply:
done